website/content/docs/import/index.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: docs
+page_title: Secrets import
+description: Secrets import allows you to safely onboard secrets from external sources into Vault KV for management.
+---
+
+
+# Secrets import
+
+<EnterpriseAlert product="vault" />
+
+@include 'alerts/beta.mdx'
+
+Having collections of secrets across various external systems can present several challenges to an organization, such as
+increased operational overhead, a greater risk of secrets sprawl, and more. Vault can offer a single source of truth (SSOT)
+solution to managing these secrets, providing increased security and reducing that overhead, yet the effort required
+to migrate all your secrets from multiple external sources into Vault can be steep. To aid in this onboarding process,
+secrets import provides a mechanism to automate it. 
+
+A secret that is imported from an external system is stored in Vault's KVv2 Secrets Engine. The import process is defined 
+an import plan, which is an HCL file. Using HCL enables users to declaratively configure which of their secrets will be read
+from an external source and where exactly they will be stored in Vault.
+
+There are three main types of HCL blocks used to configure a secrets import plan. They are source blocks, the Vault destination
+block, and mapping blocks.
+
+## Vault destinations
+
+Secrets that get imported are stored into a Vault KVv2 Secrets Engine mount. The destination will accept the mount path of
+the desired KVv2 mount and optionally a namespace. This specifies the exact location to store the secrets in. A destination
+is defined by an HCL block in the format of `destination_vault`.
+
+### HCL syntax
+
+#### Argument reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Vault destination block used to internally reference this destination inside of a mapping.
```

Comment:
Any reason we are not using the usual syntax of `name (string: <required>)`? Same question for all HCL fields.

Reply:
I was mimicking the Terraform provider docs style for how they list fields inside of resource blocks, since these aren't API fields. Not married to my idea but figured it makes sense to start with.